utils.js

```diff
@@ -44,7 +44,7 @@ exports.readFromPackageJson = function readFromPackageJson(options) {
     var packageJson;
     try {
         var fileName = options.fileName || 'package.json';
-        var packageJsonString = fs.readFileSync(path.join(process.cwd(), './' + fileName), 'utf8');
+        var packageJsonString = fs.readFileSync(path.resolve(process.cwd(), fileName), 'utf8');
```

Comment:
Couldn't this just be `fs.readFileSync(path.resolve(fileName), 'utf8')`?
"If after processing all given path segments an absolute path has not yet been generated, the current working directory is used." - https://nodejs.org/api/path.html#path_path_resolve_paths

Reply:
It could be, actually. You're right.